layers/core_checks/cc_ray_tracing.cpp

```diff
@@ -409,7 +409,8 @@ bool CoreChecks::ValidateAccelerationBuffers(VkCommandBuffer cmd_buffer, uint32_
                     }
                     if (info_loc.function == Func::vkCmdBuildAccelerationStructuresKHR &&
                         info.mode == VK_BUILD_ACCELERATION_STRUCTURE_MODE_UPDATE_KHR) {
-                        if (const auto src_as_state = Get<vvl::AccelerationStructureKHR>(info.srcAccelerationStructure)) {
+                        if (const auto src_as_state = Get<vvl::AccelerationStructureKHR>(info.srcAccelerationStructure);
+                            src_as_state->HasBuildInfo()) {
                             if (geom_i < src_as_state->build_range_infos.size()) {
```

Comment:
```suggestion
                            if (src_as_state->HasBuildInfo() && geom_i < src_as_state->build_range_infos.size()) {
```

I rather not have a `;` and multiple checks like that inside an `if` statement